data/json/mapgen/cave.json

```diff
@@ -479,7 +479,7 @@
         "............,...........",
         "............>..........."
       ],
-      "terrain": { ".": "t_rock", ",": "t_rock_floor", "W": "t_rock_floor", ">": "t_slope_up", "~": "t_water_dp" },
+      "terrain": { ".": "t_soil", ",": "t_dirt", "W": "t_dirt", ">": "t_slope_up", "~": "t_water_dp" },
```

Comment:
`t_dirt` here should be changed to `t_dirt_underground` as it's an underground section of the cave with a "roof" above.